find_row_cols_metrix.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+
+
+def find_size_matrix(matrix):
```

Comment:
The name could be confusing,  in numpy size represents the amount of elements in a numpy array not the amount of rows and collumns